lib/authority/controller.rb

```diff
@@ -144,6 +148,15 @@ def authority_forbidden(error)
       render :file => Rails.root.join('public', '403.html'), :status => 403, :layout => false
     end
 
+    # Log successful authorization attempt if enabled in the authority configuration.
+    #
+    # This method can be overloaded inside the application controller, similar to authority_forbidden.
+    def authority_success(user, action, resource)
+      if Authority.configuration.log_success
+        Authority.logger.info("#{user} successfully performed the #{action} action to this resource: #{resource}")
```

Comment:
A couple of thoughts:

- We don't actually know that they successfully performed the action, only that they successfully made it through the authorization step. We should log something like "user was authorized to perform the #{action} to the resource #{resource}"
- Is there a good reason to make this a method the user can override? `authority_forbidden` shows `403` and some people might want to show `404` or something else, so it probably needs to be overridable. But it seems like `Authority.enforce` could just say "if the `log_attempts` option is true, log the attempt."